### DISCUSSION TITLE
settings

### DISCUSSION THREAD

**Discussion by Rhyswatsonn** [jeffaroundtheworld/2024gamecollab#8] [Polls]
some random setting/location ideas

**Comment by kodriq** [comment]
prison hospital or school might be easiest cause they give a pretty clear image of what it might look like and r all closed in environments
(like we have all likely been in those or seen them on tv a lot)

**Reply by ZachLife** [reply]
how about a boarding school that way it would make sense to have a hospital incorporated into the school (either a wing or a separate building entirely) and maybe catacombs underneath

**Reply by Rhyswatsonn** [reply]
i think this would be hella cool

**Reply by kodriq** [reply]
that would be cool, and it could open up ideas for a plot like "disguised as a cool but instead they experiment on the students" or smth
